indico/modules/events/client/js/reviewing/components/ChangesConfirmation.jsx

```diff
@@ -0,0 +1,66 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {Form as FinalForm} from 'react-final-form';
+import {Form, Message, Divider} from 'semantic-ui-react';
+import {FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import './ChangesConfirmation.module.scss';
+
+export default function ChangesConfirmation() {
+  const confirmChanges = formData => {
+    console.log(formData);
+  };
+
+  return (
+    <div>
+      <Divider />
+      <FinalForm onSubmit={confirmChanges}>
+        {fprops => (
+          <Form id="confirmation-form" onSubmit={fprops.handleSubmit}>
+            <Message
+              info
+              size="tiny"
+              icon="exclamation circle"
+              header={Translate.string('Confirmation required')}
+              content={Translate.string(
+                'Editor uploaded changes to your paper. Please download and confirm. You can \
```

Comment:
I guess "Editor" will be replaced with their name?

Reply:
Can we avoid having to wrapping the string here with a `\`? It's super ugly.. :/

Maybe `<Translate>..,</Translate>` works here as well?

Reply:
yeah, i think it would be nice to replace it with their name later